index.html

```diff
@@ -6,7 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./style.css">
-    <title>Document</title>
```

Comment:
please add appropriate title

Reply:
> please add appropriate title

- [ ] As the previous reviewer requested, please add a reason as to `why?` you are requesting a change. This will enable the other person understand you better.